### PR TITLE
Error pages

### PR DESCRIPTION
**Pull request by BryanGK**
This PR is for 2 new error pages, 404 and 500.

### CODE REVIEW

**Review by wenzowski — CHANGES_REQUESTED** [review]
Looking great–can we demo our pages in dev environment to product owner and get approval before merging?

**Comment by BryanGK** [comment]
@wenzowski sure no problem. I'll just need to re-deploy this to dev. Give me a couple mins.

**Review by wenzowski — APPROVED** [review]
✨

**Pull request merged**
